Automattic/jetpack
# Star Rating Block: allow 0 stars to be selected.

Fixes #18925

#### Changes proposed in this Pull Request:
If the first star is clicked when the rating is 0.5 stars, update the rating to 0 stars.

#### Jetpack product discussion
* n/a

#### Does this pull request change what data or activity we track or use?
* no

#### Testing instructions:
1. Install, activate, and connect Jetpack.
2. Create a new post and add a Rating block.
3. Click on the first star multiple times. The star should change from 1 star to 0.5 stars to 0 stars, then back to 1 star.
4. Try different changing from different star ratings and make sure the behavior is correct.
5. Set a star rating block to 0 stars and publish the post. Make sure that the published block displays correctly.


#### Proposed changelog entry for your changes:
* Rating Block: allow 0 stars to be selected.
